Dockerfile

```diff
@@ -4,6 +4,9 @@ LABEL maintainer="Chris Kankiewicz <Chris@ChrisKankiewicz.com>"
 # Define Mumble version
 ARG MUMBLE_VERSION=1.3.0
 
+ENV CONFIG_PATH=/etc/mumble/config.ini \
+    SUPERUSER_PASSWORD=
```

Comment:
Can we remove the blank `SUPERUSER_PASSWORD` declaration?

Reply:
Sure